docs/upgrade/README.md

```diff
@@ -15,6 +15,51 @@ for each version you are upgrading across.
 Note that we do not support downgrading FlowFuse to previous levels once an upgrade
 has been performed.
 
+### Upgrading to 1.14
+
+Together with new application features, this release updates the Helm sub-chart, Bitnami's Postgresql, version.
+If local PostgreSQL database instance is used, upgrading to this version, using our Helm chart, requires additional steps.
+
+1. Backup the database
+
+   For linux:
+      
+      ```bash
+      kubectl run -it --rm db-backup --env=“PGPASSWORD=$putPasswordHere” --image ubuntu/postgres:14-22.04_edge -- bash -c “pg_dump -h flowforge-postgresql -U forge flowforge” | head -n -2 > db.sql
```

Comment:
We might need to include details of where to find the PG password for the `forge` user